server/src/main/resources/org/elasticsearch/common/reference-docs-links.txt

```diff
@@ -0,0 +1,44 @@
+INITIAL_MASTER_NODES                           important-settings.html#initial_master_nodes
+DISCOVERY_TROUBLESHOOTING                      discovery-troubleshooting.html
+UNSTABLE_CLUSTER_TROUBLESHOOTING               troubleshooting-unstable-cluster.html
+LAGGING_NODE_TROUBLESHOOTING                   troubleshooting-unstable-cluster.html#troubleshooting-unstable-cluster-lagging
+SHARD_LOCK_TROUBLESHOOTING                     troubleshooting-unstable-cluster.html#troubleshooting-unstable-cluster-shardlockobtainfailedexception
+NETWORK_DISCONNECT_TROUBLESHOOTING             troubleshooting-unstable-cluster.html#troubleshooting-unstable-cluster-network
+CONCURRENT_REPOSITORY_WRITERS                  diagnosing-corrupted-repositories.html
+ARCHIVE_INDICES                                archive-indices.html
+HTTP_TRACER                                    modules-network.html#http-rest-request-tracer
+LOGGING                                        logging.html
+BOOTSTRAP_CHECK_HEAP_SIZE                      bootstrap-checks-heap-size.html
+BOOTSTRAP_CHECK_FILE_DESCRIPTOR                bootstrap-checks-file-descriptor.html
+BOOTSTRAP_CHECK_MEMORY_LOCK                    bootstrap-checks-memory-lock.html
+BOOTSTRAP_CHECK_MAX_NUMBER_THREADS             max-number-threads-check.html
+BOOTSTRAP_CHECK_MAX_FILE_SIZE                  bootstrap-checks-max-file-size.html
+BOOTSTRAP_CHECK_MAX_SIZE_VIRTUAL_MEMORY        max-size-virtual-memory-check.html
+BOOTSTRAP_CHECK_MAXIMUM_MAP_COUNT              bootstrap-checks-max-map-count.html
+BOOTSTRAP_CHECK_CLIENT_JVM                     bootstrap-checks-client-jvm.html
+BOOTSTRAP_CHECK_USE_SERIAL_COLLECTOR           bootstrap-checks-serial-collector.html
+BOOTSTRAP_CHECK_SYSTEM_CALL_FILTER             bootstrap-checks-syscall-filter.html
+BOOTSTRAP_CHECK_ONERROR_AND_ONOUTOFMEMORYERROR bootstrap-checks-onerror.html
```

Comment:
The max length for padding seems like it will get overrun. Can we start with larger padding to limit having to reformat this file in the future?

Reply:
Fair point. Increased to 64 because that ought to be enough for anybody ;)